comp/metadata/host/hostimpl/utils/host_nix_test.go

```diff
@@ -26,6 +26,9 @@ func TestGetHostInfo(t *testing.T) {
 	info := GetInformation()
 	expected, err := host.Info()
 	require.NoError(t, err)
+	// boot time can be computed dynamically using uptime on some platforms, in which case
+	// there can be an off-by-one error
+	assert.InDelta(t, expected.BootTime, info.BootTime, 1.5)
 	assert.Equal(t, expected.BootTime, info.BootTime)
```

Comment:
I'd assume you want this line gone if you added a delta comparison?

Reply:
Oh yeah definitely, nice catch !